### PR TITLE
Enable jasmine again for running on Travis.

### PR DESCRIPTION
**Pull request by nisevi**
https://www.pivotaltracker.com/story/show/152513262

### CODE REVIEW

**Comment by nisevi** [comment]
@tansaku @vardyb @sigu the errors that we have are:

image

**Comment by tansaku** [comment]
hmm, at least those are new ...

**Comment by tansaku** [comment]
I get this error trying to run locally:

```
ERROR Sprockets::Rails::Helper::AssetNotPrecompiled: Asset was not declared to be precompiled in production.\nAdd `Rails.application.config.assets.precompile += %w( jasmine-jquery.js )` to `config/initializers/assets.rb` and restart your server
	/Users/tansaku/.rvm/gems/ruby-2.4.2/gems/sprockets-rails-3.2.1/lib/sprockets/rails/helper.rb:363:in `raise_unless_precompiled_asset'
```

**Comment by tansaku** [comment]
can you run locally @nisevi ?

**Comment by nisevi** [comment]
@tansaku working on it now :+1:

**Comment by nisevi** [comment]
@tansaku yup I'm getting the same issues locally. I can run `jasmine:ci` locally without problems.

**Comment by nisevi** [comment]
@tansaku the problem is here:

```
describe('gem bootstrap_sortable_rails', function() {
    var header;
    beforeEach(function() {
        setFixtures('<table class="sortable"><thead><tr><th data-sortkey="0">Name</th></tr></thead>' +
            '<tbody><tr><td data-value="Harrow Baptist Church"><a href="/organisations/1">Harrow Baptist Church</td></tr>' +
            '<tr><td data-value="Human Touch Worldwide"><a href="/organisations/1">Human Touch Worldwide</td></tr></tbody></table>');
        header = $('th');
    });
    describe('when clicking on the tableheader', function() {
        it('sorts A-Z then Z-A', function() {
            header.click();
            expect($('td').first()).toHaveText('Harrow Baptist Church');
            header.click();
            expect($('td').first()).toHaveText('Human Touch Worldwide')
        });
        it('inserts an arrow after click', function() {
            header.click();
            expect(header.find('span')).toHaveClass('arrow');
        });
    });
});
```

more precisely in the `before each` function.

File: spec/javascripts/bootstrap_sortable_rails_spec.js [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
+        appendSetFixtures('</tbody></table>')
         header = $('th');
     });
     describe('when clicking on the tableheader', function() {
         it('sorts A-Z then Z-A', function() {
             header.click();
             expect($('td').first()).toHaveText('Harrow Baptist Church');
             header.click();
-            expect($('td').first()).toHaveText('Human Touch Worldwide')
+            expect($('td').last()).toHaveText('Human Touch Worldwide')
```
Comment (nisevi) [inline spec/javascripts/bootstrap_sortable_rails_spec.js]: @tansaku I did this change just to check that the problem was the sorting functionality, and yes, when we click the header the sorting isn't working :+1: .

Comment (nisevi) [reply spec/javascripts/bootstrap_sortable_rails_spec.js]: The functionality on the website is working, the one that is failing to mimic the behavior is the test itself.

**Pull request merged**